src/Customers.php

```diff
@@ -51,6 +51,12 @@ public static function createCustomer($params = [])
     {
         $requiredParams = ['reference_id', 'given_names'];
 
+        if (array_key_exists('mobile_number', $params)) {
+            array_push($requiredParams, 'mobile_number');
+        } else {
+            array_push($requiredParams, 'email');
+        }
```

Comment:
hi @glendaesutanto sorry but I think the logic here is incorrect. so if one of the field doesn't exist in the param we need to make sure that the other does right?

so it should look something like this cmiiw:
```suggestion
        if (!array_key_exists('mobile_number', $params)) {
            array_push($requiredParams, 'email');
        }
        
        if (!array_key_exists('email', $params)) {
            array_push($requiredParams, 'mobile_number');
        }
```

Reply:
@nazieb Ah yeah I think yours is correct and easier to understand... I'll change it.

For my logic, I want to give a brief explanation about it. I think of the "as long as it is enough to make it valid" way... If you don't mind and have time, can you give me a counterexample regarding my logic here? I just want to know because I haven't thought of it.
Cases that I consider:
- `mobile_number` and `email` exist
- `mobile_number` exists and `email` doesn't exist
- `mobile_number` doesn't exist and `email` exists
- `mobile_number` and `email` don't exist
Are there any cases that I miss or do I not have a perfect understanding regarding this matter? Thank you so much for your help!

Reply:
@glendaesutanto aah yes, I get your logic now, thanks for explaining! and yes I think it fulfills all the cases that you mentioned.

however I must admit that it's a bit hard to scan, or maybe it's just my slow brain 😄
but anyway, it's LGTM! thanks!

Reply:
@nazieb noo, I do realize now it's a bit hard to understand, I'm so sorry :sweat_smile: 
anyway thank you for your review :smile: